pinot-spi/src/main/java/org/apache/pinot/spi/config/table/TimestampIndexGranularity.java

```diff
@@ -104,6 +104,6 @@ public static FieldSpec getFieldSpecForTimestampColumnWithGranularity(FieldSpec
    * @return Time conversion expression
    */
   public static String getTransformExpression(String timeColumn, TimestampIndexGranularity granularity) {
-    return "dateTrunc(\'" + granularity + "\', " + timeColumn + ")";
+    return String.format("dateTrunc('%s', %s)", granularity, timeColumn);
```

Comment:
I remember @richardstartin mentioned that string plus is faster than string.format.